### PR TITLE
FOIA-90: VI.C (5) and VII.E -- "ten oldest" rules

### PR DESCRIPTION
**Pull request by rgristroph**

### CODE REVIEW

File: docroot/modules/custom/foia_ui/js/foia_ui.validation.js [hunk 1]
```diff
@@ -15,6 +15,17 @@
         return value <= Number(target.val());
       }, "Please enter a lesser value." );
 
+      // lessThanEqualToNA
+      $.validator.addMethod( "lessThanEqualToNA", function( value, element, param ) {
+        var target = $( param );
+        // Treat N/A like 0.
+        if ( value == "N/A" || value == "n/a" )
+          value = 0;
```
Comment (amanire) [inline docroot/modules/custom/foia_ui/js/foia_ui.validation.js]: I think this is fine functionally, but [coding style-wise](https://www.drupal.org/docs/develop/standards/javascript/javascript-coding-standards#structures):
>Control structures MUST always use curly braces, even in situations where they are optional. Having them increases readability and decreases the likelihood of logic errors being introduced when new lines are added.

Comment (amanire) [reply docroot/modules/custom/foia_ui/js/foia_ui.validation.js]: `if ( value.toLowerCase() == "n/a" )` would be a more concise expression

File: docroot/modules/custom/foia_ui/js/foia_ui.validation.js [hunk 2]
```diff
@@ -15,6 +15,17 @@
         return value <= Number(target.val());
       }, "Please enter a lesser value." );
 
+      // lessThanEqualToNA
+      $.validator.addMethod( "lessThanEqualToNA", function( value, element, param ) {
+        var target = $( param );
+        // Treat N/A like 0.
+        if ( value == "N/A" || value == "n/a" )
```
Comment (amanire) [inline docroot/modules/custom/foia_ui/js/foia_ui.validation.js]: `if ( value.toLowerCase() == "n/a" )` would be a more concise expression

**Comment by bemarlan** [comment]
@rgristroph ^

File: docroot/modules/custom/foia_ui/js/foia_ui.validation.js (hunk 1 above)
Comment (rgristroph) [reply docroot/modules/custom/foia_ui/js/foia_ui.validation.js]: Fixed

Comment (rgristroph) [reply docroot/modules/custom/foia_ui/js/foia_ui.validation.js]: Thanks - fixed

File: docroot/modules/custom/foia_ui/js/foia_ui.validation.js (hunk 2 above)
Comment (rgristroph) [reply docroot/modules/custom/foia_ui/js/foia_ui.validation.js]: I had to wrap target and value in "String()", but that works.

**Comment by rgristroph** [comment]
I think this should address everything and also add section VII.E.

**Pull request merged**